web/src/features/panels/zone/EstimationCard.tsx

```diff
@@ -23,7 +42,17 @@ export default function EstimationCard({
       return <AggregatedCard estimatedPercentage={estimatedPercentage} />;
     }
     case 'estimated': {
-      return <EstimatedCard estimationMethod={estimationMethod} />;
+      return (
+        <div>
+          <EstimatedCard
+            estimationMethod={estimationMethod}
+            setCollapsedNumber={setCollapsedNumber}
+          />
+          {currentlyShowingFeedbackCard && (
```

Comment:
Nitpick but it's quite nice to use the pattern isXVisibile for visibilty naming so isFeedbackCardVisibile in this case